static/styles/popovers.css

```diff
@@ -351,6 +358,59 @@ ul {
             margin: 10px 0;
         }
     }
+
+    .tab-data {
```

Comment:
I tweaked this to add a 20px bottom margin to this container, since it looked bad when scrolled to the bottom.